### PR TITLE
Consolidate root-mean-square functions

### LINKED ISSUE #103: Move RMSF into pmda.rms
Need to create an rms module as in MDAnalysis.analysis.rms for RMSF. Instead of having pmda.rmsf all in one file, we will make a submodule:

`rms/__init__.py`
    `rmsd.py`
    `rmsf.py`

and `in __init__.py` we will have:

```python
from __future__ import absolute_import`
from .rmsf import RMSF
from .rmsd import RMSD

__all__ = ["RMSD", "RMSF"]
```

This will require doc restructuring.

### PR DESCRIPTION
**Pull request by nawtrey**
Moves all root-mean-square calculations (currently RMSD and RMSF) into their own module, updates documentation, tests, and .rst files to match new structure.

Fixes #103 

Changes:
 - Create pmda/rms
 - Rename and move rms.py to pmda/rms.rmsd.py
 - Add example to RMSD documentation in /rms.rmsd.py
 - Move rmsf.py into /rms
 - Fix imports in tests and .rst files
 - Correct documentation example imports

PR Checklist
------------
 - [x] Tests?
 - [x] Docs?
 - [x] CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

File: pmda/rms/__init__.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+"""
+pmda.rms
+========
+
+Ready to use root-mean-square analyses built via MDAnalysis with dask.
+
+The full documentation can be found at https://www.mdanalysis.org/pmda/.
+
+Note that by default no modules are imported in the top-level name
```
Comment (orbeckst) [inline pmda/rms/__init__.py]: I don't understand this sentence. We _are_ importing `RMSD` and `RMSF` into `rms` in the lines below (which is how it should be).

File: pmda/test/test_rmsd.py [hunk 2]
```diff
@@ -6,16 +6,14 @@
...
 
+from __future__ import absolute_import, division, print_function
 import pytest
 from numpy.testing import (assert_almost_equal, assert_array_equal,
                            assert_array_almost_equal)
-
 import MDAnalysis
 from MDAnalysisTests.datafiles import (PSF, DCD)
-
-from pmda import rms
```
Comment (orbeckst) [inline pmda/test/test_rmsd.py]: This line should still work, together with `rms.RMSD()`, right?

I  generally like imports that keep a name space attached to objects – makes it a lot easier to see where stuff comes from. I am ok with using `from pmda.rms import RMSD` here, just want to confirm that `rms.RMSD` will remain to work.

File: pmda/rms/__init__.py (hunk 1 above)
Comment (nawtrey) [reply pmda/rms/__init__.py]: As far as I can tell, this is an artifact carried over from MDAnalysis that I didn't remove. I'll just take it (and the following lines) out.

File: pmda/test/test_rmsd.py (hunk 2 above)
Comment (nawtrey) [reply pmda/test/test_rmsd.py]: Yes, you can still do 

`from pmda import rms`
`ca = universe.select_atoms("name CA")`
`universe.trajectory.rewind()`
`rmsd1 = rms.RMSD(ca, ca).run()`

**Comment by orbeckst** [comment]
Sorry @nawtrey – totally fell off my radar.

**Pull request merged**